api/app/src/main/resources/db/migration/V20241219__add_description_to_packet_table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE packet
+    ADD COLUMN description TEXT;
```

Comment:
This raises the question of how to back-fill existing packets' descriptions.

Reply:
Something like this, in the PacketService?

```kt
fun backfillPacketDescriptions() {
    val packets = packetRepository.findAll()
    val updatedPackets = packets.map { packet ->
        val metadata = outpackServerClient.getMetadataById(packet.id)
        packet.description = getDescriptionForPacket(metadata)
        packet
    }
    packetRepository.saveAll(updatedPackets)
}
```
Then ssh into each production instance and invoke this function somehow, and then remove the function from the codebase